### PR TITLE
BFC fonts dithered workaround - update readme

### PR DESCRIPTION
**Pull request by RipperDrone**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +108,10 @@ Dont forget to add your user into dialout group "sudo usermod -aG dialout YOUR_U
 
 If you have 3D model animation problems, enable "Override software rendering list" in Chrome flags chrome://flags/#ignore-gpu-blacklist
 
+### Windows users
+
+If you experience smudged/dithered fonts issues, try invoking the .exe executable file with the --disable-gpu switch. This will switch off hardware graphics acceleration. Likewise, setting your graphics card setting FXAA (Antialiasing) to OFF might be a remedy as well.
```
Comment (mikeller) [inline README.md]: I think 'FXAA' is proprietary to one GPU manufacturer (Nvidia?), so this should probably be mentioned.

**Comment by mikeller** [comment]
Can you please rebase and squash into one commit?

**Comment by Anisotropic** [comment]
What's the issue with dithered fonts? Maybe it could be solved with some CSS magic?

**Comment by mikeller** [comment]
@Anisotropic: The issue is some versions of NW.js not working properly with some GPU drivers (as evidenced by the fact that disabling hardware acceleration fixes the problem). Dithered fonts are one symptom, problems with them 3D model another one, and general display weirdness like #1748 yet another one. So I suspect CSS won't solve this, as it is happening at a lower level of the stack.

**Comment by RipperDrone** [comment]
@mikeller sry, can't find out how to squash/rebase using github only / no special environment. Tried to google it, cannot choose btw merge and squash&rebase anywhere like shown in some tutorials... :-/

**Comment by RipperDrone** [comment]
image

**Comment by Docteh** [comment]
@RipperDrone if you'd made the changes in a branch you could squash into your own master.

I'm not sure what to suggest, you could make a branch, but git tools would be required to roll back the master. Its interesting seeing whats missing from githubs web ui.

**Comment by mikeller** [comment]
@RipperDrone:

To rebase / squash the commits in your branch / pull request into one with `git` on the command line you proceed as follows:
- update your local `master` branch to point to latest in https://github.com/betaflight/betaflight.git;
- in your branch, do a `git rebase -i master`;
- in the editor that opens, select `pick` for the first of your commits, `squash` for the rest of your commits, save & close (make sure the commit that you've selected `pick` for has got a message indicative of what the pull request is doing; you can edit the message if needed);
- push your changes with `git push --force-with-lease`;
- done. :-)

**Comment by RipperDrone** [comment]
@mikeller thank you for trying to bring me up to speed. I'm completely lost now, I'm sorry... 

- tried command-line git, just ended up in editor showing no commits
- installed Git Desktop for Windows, didn'T get me anywhere either
- then hoping GitKraken was easier to jump-start. It wasn't:

image

Giving up here - will take me a lot more time and reading to get stuff right :-(

**Comment by mikeller** [comment]
@RipperDrone: I suspect the problem you are facing is that you have is caused by the fact that your pull request is on the `master` branch of your clone - this is 'supported', but has a number of problems. One of them is that when you use `git rebase -i master`, git will try to rebase against your branch, and obviously not find any changesets that are different.

Since there is no good way to salvage this pull request (you can't change the branch a pull request is on) I have rebased it for you.

**Comment by Anisotropic** [comment]
> Dithered fonts are one symptom, problems with them 3D model another one, and general display weirdness like #1748 yet another one. 

I've seen some sort of "dithered fonts" on one of my job projects. And we cured them by css. It could be different causes though.

**Comment by Docteh** [comment]
@Anisotropic Was it running in a Web Browser like Google Chrome, Edge, Safari?

Or was it running in a separate thing like NWJS, flutter, electron?

**Comment by Anisotropic** [comment]
@Docteh It was running chrome.

**Comment by mikeller** [comment]
@Docteh: I've had instances of display weirdness with some versions of Chrome that were caused by running on an exotic GPU with fringe-usable drivers - normally this can be cured by disabling GPU acceleration. So NW.js and Chrome seem to be acting very similar in this respect.

**Pull request merged**